frontend/packages/client/src/pages/Proposal.js

```diff
@@ -99,7 +104,12 @@ export default function ProposalPage() {
     error: strategiesError,
   } = useVotingStrategies();
 
-  const isAdmin = proposal && proposal?.creatorAddr === user?.addr;
```

Comment:
creator of proposal was able to cancel it, now this has switch to the role the user has within the community.